source/codegen/metadata/nixnetsocket/enums.py

```diff
@@ -219,18 +219,6 @@
             },
         ]
     },
-    'ProtocolFamily': {
```

Comment:
Note: This could be used for `nxsocket`'s `domain` argument. But we have that using `AddressFamily` enum which has the same values and are used interchangebly in the implementations I've seen. Also, our documentation mentions the address family (nxAF_INET) for that argument and not the protocol family: https://github.com/ni/grpc-device/wiki/NI-XNET-Sockets-Functions#inputs-22